ChangeLog.md

```diff
@@ -40,66 +40,85 @@ Notable changes to the FreeOrion project will be documented in this file.
+    - Made droplists scroll to keep the selected row visible when navigating with the keyboard.
+    - Made numpad keys work with droplist navigation.
+    - Made current item visible when drop down list opens.
+    - Fixed formatting and layout in the save file dialog.
+    - Changed default extended tooltip delay to 3.5 s.
+    - Added a tooltip to fleet summary icon at top of screen.
+    - Added option to show IDs after object names in GUI.
+    - Made mousewheel manipulate credits scrolling.
+    - Fixed droplists falling outside the application window.
+    - Made some scrollable GUI widgets respond to keypresses to scroll their contents.
+    - Made ListBox header show for empty list boxes.
+    - Paused music when the application loses focus.
+    - Closed credits if the app is resized.
+    - Update FPS immediately when the option changes.
+    - Stopped save file dialog from reloading saves after window focus changes.
+    - Added dialog to ask user about waiting for savegame to complete before quitting.
+
+- Pedia
+    - Added access to pedia from intro screen.
+    - Included more pedia articles in search results
+    - Displayed identical parts as one entry in ship browse windows.
+    - Made Enter/Return KeyPress events on EncyclopediaDetailPanel set the focus to the search edit.
+    - Adjusted planet suitability report for tech-gated species.
+
+- Map Window
+    - Changed Fleets window quick close option from right click to left click
+    - Made ordering of systems in cycling commands alphabetical.
+    - Changed colours used to render some scanlines (indicating a not-visible object).
+    - Changed display of map scale distance to integers.
+    - Added an inner system circle on the galaxy map, which is drawn for systems with colonies.
+
+- Sidepanel
+    - Added automatic selection of ships for bombarding.
+    - Added ability to rename systems in SidePanel. System sorting considers the system's id to disambiguate if necessary.
+    - Displayed source planet in growth special accounting labels.
+    - Added indication of defending troops to invade button.
+    - In sidepanel, displayed tooltip not only for the selected focus, but also for available focus settings in the focus dropdown list.
+
+- Fleet Window
+    - Added more and more flexible indicator icons to fleet and ship panels.
+    - Labeled fleets window as "Near System X" for fleets in transit.
+    - Showed empire name in fleets window for fleets in transit.
+    - Added right-click menu command to split from a fleet ships that have less than full fighter complement.
+
+- SitRep Window
+    - Allowed SitRep right-click menu from majority of sitrep area.
+    - Added Copy to sitrep right-click context menu.
+    - Added link to help article on sitrep entry right click menu.
+    - Added right click context to hide/show types of SitReps.
+    - Changed combat log to use empire color to designate destroyed objects.
```

Comment:
The latter, there was only one log change that I noticed and I meant to make this Sitrep / Combat log but forgot, as you've found other typos I missed while copy/pasting I'll do another pass in a bit.